src/connectors/snowflake/trulens/connectors/snowflake/connector.py

```diff
@@ -71,12 +75,16 @@ def __init__(
             snowpark_session,
             init_server_side,
             init_server_side_with_staged_packages,
+            init_sis_dashboard,
             database_redact_keys,
             database_prefix,
             database_args,
             database_check_revision,
             connection_parameters,
         )
+        self.snowpark_session = snowpark_session
+        self.connection_parameters = connection_parameters
```

Comment:
why are you keeping these things, you don't seem to use them.

Reply:
I end up using them [here](https://github.com/truera/trulens/pull/1648/files#diff-7584d30e434ba8c191f76c22f3658cc4c7245f412b9672beb84134632188a0cbR344) though this implementation could be cleaner